### ISSUE TITLE
Question - Autocomplete HTML in DropDown list

### ISSUE THREAD

**Issue by ivancavlek** [Megabit/Blazorise#2969]
Hi,

Could you tell me if it's possible (and how) to include HTML and/or other Blazorise components in Autocomplete DropDown list?

![image](https://user-images.githubusercontent.com/16662190/134861107-ea469067-6a02-4171-a7d2-182b928f18df.png)

E.g. that I could have a flag or an image next to a country and that I could make data in this whole selection a bit more formatted like here:

![image](https://user-images.githubusercontent.com/16662190/134860407-7c3ef28f-818b-41e2-b134-03a3ba019a1b.png)

If it's not possible, could it be added as a feature, e.g. to make the DropDownItem in Autocomplete as RenderFragment or something similar?

**Comment by David-Moreira** [comment]
I guess it's a nice to have, and we could take a look at doing something similar to `NotFoundContent` RenderFragment. With access to either 
- the item or 
- both the text and value.

What do you think @stsrki ?
![AutocompleteRenderFragment](https://user-images.githubusercontent.com/22283161/136056730-bd9860ae-b4d1-4544-acb8-80990399a8d7.png)

**Comment by stsrki** [comment]
@David-Moreira Yes, I agree. Having something like `ItemTemplate` would be the best option.

**Comment by David-Moreira** [comment]
Oh I just noticed `NotFoundContent` has the wrong suffix, as Blazorise uses **Template** suffix instead for RenderFragments... :(

**Comment by stsrki** [comment]
It is not a big problem. NotFoundContent is also good name.
